Code/Tools/Android/ProjectBuilder/asset.build.gradle.in

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (c) Contributors to the Open 3D Engine Project.
+// For complete copyright and license terms please see the LICENSE at the root of this distribution.
+//
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+
+// CARBONATED -- begin
+// Play Asset Delivery Pack 'build.gradle' file
+
+plugins {
+    id 'com.android.asset-pack'
+}
+
+assetPack {
+    // Directory name for the asset pack
+    packName = '${CARBONATED_ASSET_PACK_NAME}'
```

Comment:
Let's rename these to omit "CARBONATED_.."

That'll make it easier to flow back to O3DE

Reply:
Removed